app/models/concerns/user/ldap_authenticable.rb

```diff
@@ -22,7 +22,7 @@ def ldap_get_user(attributes = {})
         safe_username = safe_username.gsub(keys, replacement)
       end
 
-      resource = joins(:account).find_by(accounts: { username: safe_username })
+      resource = joins(:account).find_by('accounts.username ILIKE ?', safe_username)
```

Comment:
I think something like this would be more efficient thanks to the `index_accounts_on_username_and_domain_lower` index:
```suggestion
      resource = joins(:account).merge(Account.where(Account.arel_table[:username].lower.eq safe_username.downcase)).take
```

Reply:
Thanks! This works equally well, and I'm sure it's faster and probably also slightly safer than `ILIKE`. :+1: